packages/datastore/src/storage/adapter/InMemoryStore.ts

```diff
@@ -12,15 +12,19 @@ export class InMemoryStore {
 	multiRemove = async (keys: string[], callback?) => {
 		keys.forEach(k => this.db.delete(k));
 
-		callback();
+		if (callback && typeof callback === 'function') {
```

Comment:
I think a `typeof` check is enough, non-truthy values will not be functions

```suggestion
		if (typeof callback === 'function') {
```

Same below